### PR TITLE
Print fastlane env log on error

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

**Review by viktorbenei — CHANGES_REQUESTED** [review]
No need for buffer, and error is not handled properly

File: main.go [hunk 1]
```diff
@@ -21,13 +21,15 @@ type ConfigsModel struct {
 	WorkDir        string
 	Lane           string
 	UpdateFastlane string
+	DeployDir      string
```
Comment (godrei) [inline main.go]: Please remove `DeployDir` at all

File: main.go [hunk 2]
```diff
@@ -257,6 +260,17 @@ func main() {
...
 	if err := cmd.Run(); err != nil {
+		fmt.Println()
+		log.Errorf("Fastlane command: (%s) failed", command.PrintableCommandArgs(true, fastlaneCmd))
+		log.Errorf("See the error log below, and use it to send issue report to fastlane github issue tracker:")
+		log.Printf("https://github.com/fastlane/fastlane/blob/master/.github/ISSUE_TEMPLATE.md#environment")
+		fmt.Println()
+
+		fastlaneEnvLogCmd := []string{"fastlane", "env"}
+		inputReader := strings.NewReader("n")
+		if err := command.RunCommandWithReaderAndWriters(inputReader, os.Stdout, os.Stdout, fastlaneEnvLogCmd[0], fastlaneEnvLogCmd...); err != nil {
```
Comment (godrei) [inline main.go]: Use `rubycommand.New()` to generate the command model.
Do not forget to:

- set the work dir
- set stdin (`n`)
- and `cmd.SetStdout(os.Stdout).SetStderr(os.Stderr)`

File: main.go [hunk 3]
```diff
@@ -257,6 +260,17 @@ func main() {
...
+			log.Warnf("Fastlane command: (%s) failed", command.PrintableCommandArgs(true, fastlaneEnvLogCmd))
```
Comment (godrei) [inline main.go]: the the generated command's instance method for printing the raw command, instead of `command.PrintableCommandArgs`

**Review by godrei — CHANGES_REQUESTED** [review]

File: main.go [hunk 4]
```diff
@@ -257,6 +257,26 @@ func main() {
 	cmd.SetDir(workDir)
 
 	if err := cmd.Run(); err != nil {
+		fmt.Println()
+		log.Errorf("Fastlane command: (%s) failed", cmd.PrintableCommandArgs())
+		log.Errorf("See the error log below, and use it to send issue report to fastlane github issue tracker:")
+		log.Printf("https://github.com/fastlane/fastlane/blob/master/.github/ISSUE_TEMPLATE.md#environment")
```
Comment (godrei) [inline main.go]: the url should be: `https://github.com/fastlane/fastlane/issues/new`

File: main.go [hunk 5]
```diff
@@ -257,6 +257,26 @@ func main() {
 	cmd.SetDir(workDir)
 
 	if err := cmd.Run(); err != nil {
+		fmt.Println()
+		log.Errorf("Fastlane command: (%s) failed", cmd.PrintableCommandArgs())
+		log.Errorf("See the error log below, and use it to send issue report to fastlane github issue tracker:")
```
Comment (viktorbenei) [inline main.go:262]: If you want to send an issue report to fastlane (https://github.com/fastlane/fastlane/issues/new), you can find the output of fastlane env (below):

**Pull request merged**